run/CESM/HEMCO_Config.rc

```diff
@@ -3224,15 +3225,17 @@ ch
 
 #==============================================================================
 # --- Offline dust emissions ---
-# NOTE: Use flag C if running in CESM
 #==============================================================================
 (((OFFLINE_DUST
-(((.not.DustDead.or.DustGinoux
-0 EMIS_DST1 $ROOT/OFFLINE_DUST/${RUNDIR_OFFLINE_DUST_DIR}/0.5x0.625/$YYYY/$MM/dust_emissions_05.$YYYY$MM$DD.nc EMIS_DST1 1980-2022/1-12/1-31/* C xy kg/m2/s DST1 - 3 2
```

Comment:
The old code uses flag C so the new entries should as well.

Reply:
Thanks @lizziel. Now fixed in 3901263a.